hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapStoreTest.java

```diff
@@ -1104,10 +1104,55 @@ public void storeAll(Map<Integer, Integer> map) {
         assertEquals(1_500, map.size());
     }
 
+    // See https://github.com/hazelcast/hazelcast/issues/21414
+    @Test(timeout = 120000)
+    public void testBugFix_21414() {
```

Comment:
I believe it would be better to add the proper test to `ExceptionUtilTest`.

Reply:
It may be more reasonable to do that. Do you think this test should stay?

Reply:
I think we don't need to have it, the map store problem is just a by-product of something else

Reply:
moved the test to ExceptionUtilTest